cypress/integration/e2e/sign-in-gate.spec.js

```diff
@@ -58,380 +60,424 @@ describe('Sign In Gate Tests', function () {
             .then(cy.wrap);
     };
 
-    describe('SignInGateMain', function () {
-        beforeEach(function () {
-            // sign in gate main runs from 0-900000 MVT IDs, so 500 forces user into test
-            setMvtCookie('500');
-
-            // set article count to be min number to view gate
-            setArticleCount(3);
+    /**
+     * AB test switches are defined in Frontend and required to be "ON" for Cypress E2E Sign In Gate Test to successfully pass.
+     * In order not to break CI builds in DCR if someone switches off a Sign In Gate AB test in the Frontend Admin tool, we run
+     * a setup test suite to load active switches from Froented into a Cypress Environment Variable available to this spec only.
+     * We use this env var to check if a switch is ON, to conditionally run relevant test suites.
+     *
+     * Warning - this means we can turn switches on/off in the Frontend Admin PROD UI without needing to make code changes to DCR,
+     * but you will need to make sure switches are ON when developing locally or in CODE.
+     *
+     * Ensure you wrap all your AB test suites with "onlyRunIf(switchIsOn('abNameOfTest', () => {})" for this to work.
+     */
+    describe('Get live Switches from Frontend', function () {
+        it('Load article and get switches', function () {
+            visitArticle();
+            cy.window().then((win) => {
```

Comment:
I have to run "cy" commands within a test, which is why there is a suite to load the live switches from Frontend.  Open to better suggestions!